### ISSUE TITLE
gun is meant to be fun and yet...

### ISSUE THREAD

**Issue by LLFourn** [LLFourn/gun#88]
it doesn't output any emojis. Right now the command logic is full of `eprintln`. I think we should replace every `eprintln` with a custom function for the purpose of the message so it can be formatted correctly.

for example, if it's warning the user about something prefix the message with the :warning: emoji would be appropriate. If something worked prefix it with :tada: and so on.

**Comment by keblek** [comment]
should this be implemented via the fmt::Display?

**Comment by LLFourn** [comment]
@keblek I don't think so. I imagine the easiest way to would be just to create a macro like this:

```rust
macro_rules! elog {
    (@warning $($tt:tt)*) => { eprint!("⚠️"); eprintln!($($tt)*);
    (@celebration $($tt:tt)*) => { eprint!("🎉"); eprintln!($($tt)*); };
    // and so on
}

// which you could then use like:

elog!(@celebration "your wallet was set up successfully at {}", path) 
```

**Comment by keblek** [comment]
#104
